packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/FlipbookViewer.js

```diff
@@ -1,17 +1,83 @@
-import React from 'react'
+import React, { useEffect, useRef, useState } from 'react'
 import { Box } from 'grommet'
 import PropTypes from 'prop-types'
+
 import locationValidator from '../../helpers/locationValidator'
+import { placeholder } from '../SingleImageViewer/hooks/useSubjectImage'
+
+import SingleImageViewer from '../SingleImageViewer/SingleImageViewer'
```

Comment:
```suggestion
import SingleImageViewer from '../SingleImageViewer/SingleImageViewer.js'
```

ES imports must be full file paths, so we should get into the habit of using those.

Importing a single JS file from `SingleImageViewer` here is dangerous, in that future refactors of the `SingleImageViewer` component might change that JS file, without realising that it's also used here, thus breaking this viewer. In the classifier, it's always safest to import a component via its `index.js` file ie. treat every folder as a module and import the complete module.